grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/db/AddressDao.java

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
- `TypeDBBeans`: Sort of. This is more about CDI beans. We have just have one method, but generally you have a class which containers producers for any beans not directly under your control. Thus, this class could potentially have producers of different beans.
- `TypeDBSessionWrapper`: I'm open to alt names, but I prefer to avoid duplicate names. In this case, I don't want to conflict with TypeDB's own `TypeDBSession`. It could be `Session`, but that doesn't tell me much. What kind of session is it?

I think adjusting the package makes sense though. I like your suggestion. Keep the two in `db` and move the daos to `db.dao`. @SullivanDaly Can you knock this out for us

Reply:
Sorry, should've seen this earlier.

OK, if these are the names that are most logical in the context of CDI and a small example, let's go with them.